matlab/schemas/+preprocess/Sync.m

```diff
@@ -30,6 +30,8 @@ function makeTuples(self, key)
                 packetLen = dat.analogPacketLen;
             end
             datT = pipetools.ts2sec(dat.ts, packetLen);
+            datT = datT-datT(1); % subtract offset
```

Comment:
`datT` is only used to get the sampling rate of the behavior clock, so the offset doesn't matter. Removed the line where I subtracted the offset.